src/reducers/counter.js

```diff
@@ -1,8 +1,5 @@
 import { createReducer } from 'utils';
-
-// normally this would be imported from /constants, but in trying to keep
-// this starter kit as small as possible we'll just define it here.
-const COUNTER_INCREMENT = 'COUNTER_INCREMENT';
+import { COUNTER_INCREMENT } from 'constants/counter';
```

Comment:
Also I know it's petty, but would you mind lining up the `from`'s in both this file and `HomeView.js`. I'd end up changing them eventually anyways, so best to just get it out of the way now. Sorry :(